processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/FieldNameConverter.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.processor;
+
+import com.google.common.base.CaseFormat;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Collection of utility functions to generate names of getters and setters, fields and other related staff for a given
```

Comment:
```suggestion
 * Collection of utility functions to generate names of getters and setters, fields and other related stuff for a given
```

Reply:
I learned the words "staff" and "stuff" many years ago when I was unable to hear a difference between them. Today I can pronounce both words correctly but I cannot remember which one is which. Do you know a mnemonics? :)

Reply:
I personally make the connection in my head with stuffing a Turkey, filling it with -yummy- stuff :)